library/Leads/Exporter/AbstractExcelExporter.php

```diff
@@ -52,16 +52,26 @@ public function export($config, $ids = null)
      */
     protected function exportWithFormat($config, $ids, $format)
     {
+        $actTime = time();
+        
+        $lastExportDate = null;
+        if ($config->onlyExportSinceLastExportDate && \Validator::isDatim($config->lastExportDate))
+        {
+          $lastExportDate = $config->lastExportDate;
+        }
+        
         $dataCollector = $this->prepareDefaultDataCollector($config, $ids);
-        $reader = new ArrayReader($dataCollector->getExportData());
+        $reader = new ArrayReader($dataCollector->getExportData($lastExportDate));
 
         if ($config->headerFields) {
             $reader->setHeaderFields($this->prepareDefaultHeaderFields($config, $dataCollector));
         }
 
         $row = new Row($config, $this->prepareDefaultExportConfig($config, $dataCollector));
 
-
+        \Database::getInstance()->prepare('UPDATE tl_lead_export SET lastExportDate = ? WHERE id = ?')
```

Comment:
Damn, missed that part. It was in my brain, but didn't found the way into the code.

Will be added asap.